frontend/src/component/Workspace/OpenShareFolderApp.jsx

```diff
@@ -61,11 +61,12 @@ export class OpenShareFolderApp extends React.Component {
     console.log('%c<OpenShareFolderApp> did Update', 'color: #dcae84', this.props)
 
     if (props.match && prevProps.match && props.match.params.idws !== prevProps.match.params.idws) {
-      props.onUpdateAppOpenedType(false)
+      props.onUpdateAppOpenedType()
```

Comment:
onUpdateAppOpenedType expects a param, why removing it ?

Reply:
This is a mistake
```suggestion
      props.onUpdateAppOpenedType(false)
```